nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -474,6 +475,16 @@
             .allowableValues("true", "false")
             .build();
 
+    public static final PropertyDescriptor SUPPORT_HTTP2_PROTOCOL = new PropertyDescriptor.Builder()
+            .name("support-http2")
+            .description("Determines whether or not to support the HTTP 2 protocol version.")
+            .displayName("Support HTTP 2")
+            .required(true)
+            .defaultValue("True")
+            .allowableValues("True", "False")
```

Comment:
Should Boolean values be lowercase strings to match standard primitive usage and other Boolean properties?

Reply:
Debatable on this :) Typically we favor capitalized words in nifi, but with boolean we've tended to use true/false instead of True/False. There's a good mixture throughout the codebase unfortunately. If there is already at least one boolean property in the processor i would say at least try to stay consistent. Otherwise... 🤷‍♂️ 
Of note, though, there's no need for the Boolean Validator, as there are allowable values.

Reply:
Thanks for the reply, it looks like this processor has both sets of options, so with that background, the current approach works.

Reply:
Yes it seems like even just within this processor there is both "true" and "True". I'll remove the boolean validator.